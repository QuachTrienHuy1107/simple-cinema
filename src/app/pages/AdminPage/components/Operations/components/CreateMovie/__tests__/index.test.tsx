import * as React from "react";
import { render } from "@testing-library/react";

import { CreateMovie } from "..";

jest.mock("react-i18next", () => ({
    useTranslation: () => {
        return {
            t: str => str,
            i18n: {
                changeLanguage: () => new Promise(() => {}),
            },
        };
    },
}));

describe("<CreateMovie  />", () => {
    it("should match snapshot", () => {
        const loadingIndicator = render(<CreateMovie />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
