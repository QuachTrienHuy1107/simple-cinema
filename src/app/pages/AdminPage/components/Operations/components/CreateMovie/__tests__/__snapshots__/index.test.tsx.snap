// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateMovie  /> should match snapshot 1`] = `
.c1 {
  line-height: 0;
  font-weight: 600;
  border-radius: 300px;
  display: inline-block;
  outline: none;
  border: none;
  font-size: 1rem;
  -webkit-transition: all 0.5s;
  transition: all 0.5s;
}

.c1:hover {
  box-shadow: 0px 10px 15px 0px rgba(59,55,188,0.5);
}

.c0 .ant-tabs-content-holder,
.c0 .ant-tabs-nav {
  background-color: #151f30;
  padding: 20px 40px;
  border-radius: 16px;
  margin-bottom: 30px;
  border-bottom: none;
}

.c0 .ant-tabs-content-holder::before,
.c0 .ant-tabs-nav::before {
  border: none;
}

<div
  class=""
>
  <div
    class="c0"
  >
    <div
      class=""
    >
      <button
        class="ant-btn ant-btn-round ant-btn-lg c1 icon-button"
        type="button"
      >
        <span>
          Create
        </span>
      </button>
    </div>
  </div>
</div>
`;
